src/main/java/hudson/plugins/parameterizedtrigger/BlockableBuildTriggerConfig.java

```diff
@@ -46,46 +53,57 @@ public BlockingBehaviour getBlock() {
     }
 
     @Override
-    public List<QueueTaskFuture<AbstractBuild>> perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListener listener) throws InterruptedException, IOException {
+    public List<QueueTaskFuture<AbstractBuild>> perform(AbstractBuild<?, ?> build, Launcher launcher,
+            BuildListener listener) throws InterruptedException, IOException {
         List<QueueTaskFuture<AbstractBuild>> r = super.perform(build, launcher, listener);
-        if (block==null) return Collections.emptyList();
+        if (block == null)
+            return Collections.emptyList();
         return r;
     }
 
     @Override
-    public ListMultimap<AbstractProject, QueueTaskFuture<AbstractBuild>> perform2(AbstractBuild<?, ?> build, Launcher launcher, BuildListener listener) throws InterruptedException, IOException {
-        ListMultimap<AbstractProject, QueueTaskFuture<AbstractBuild>> futures = super.perform2(build, launcher, listener);
-        if(block==null) return ArrayListMultimap.create();
+    public ListMultimap<AbstractProject, QueueTaskFuture<AbstractBuild>> perform2(AbstractBuild<?, ?> build,
+            Launcher launcher, BuildListener listener) throws InterruptedException, IOException {
+        ListMultimap<AbstractProject, QueueTaskFuture<AbstractBuild>> futures = super.perform2(build, launcher,
+                listener);
+        if (block == null)
+            return ArrayListMultimap.create();
```

Comment:
Again, best to just revert any change that does not further the stated goal of the PR.